plugin.video.mytv_bg/resources/settings.xml

```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
-<settings>
-    <category label="Login information">
-        <setting id="username" type="text" label="30200" default="" />
-        <setting id="password" type="text" option="hidden" label="30210" enable="!eq(-1,)" default="" />
-    </category>
-    <category label="Server">
-    	<setting id="server_url" type="text" label="5000" default="" />
-    </category>
-    <category label="General">
-        <setting id="force_viewmode" type="bool" label="30010" default="true"/>
-        <setting id="update_on_pageswitch" type="bool" label="30011" default="true"/>
-    </category>
-</settings>
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<settings>
+    <category label="Login information">

```

Comment:
It would also be cool if you could use localized strings in the categories as well instead of hardcoding english strings